vignettes/osmdata.Rmd

```diff
@@ -49,11 +49,11 @@ An `osmdata` query consists of an initial `opq()` query and one or more features
 q <- opq (bbox="greater london uk")
 q <- add_feature (q, key="highway", value="motorway")
 ```
-The corresponding data may then be extracted as `R Spatial` objects with,
+The corresponding data may then be extracted as a `Spatial` object defined by the `sp` package, with,
 ```{r osmdata_sp}
 osmdata_sp (q)
 ```
-or an `R Sinple Features` objects with
+or an `R Simple Features` objects with
```

Comment:
That way the data is useful regardless of if you use R for analysis or not, as shown here:

![osm-qgis](https://cloud.githubusercontent.com/assets/1825120/23326333/7189dd04-faf4-11e6-91f1-bd1b0c366cad.png)